src/components/Schools/EnterStudentDetails.js

```diff
@@ -0,0 +1,74 @@
+
+import React from 'react';
+import { useState, useEffect } from 'react'
+import styles from '../../styles/ExpandedVideo.module.css'
+import Image from 'next/image'
+
+
+import { Amplify, API } from 'aws-amplify';
+import awsconfig from '../../aws-exports';
+import * as mutations from "../../graphql/mutations";
+import * as queries from '../../graphql/queries';
+
+
+const EnterClass = React.memo(({classID, onClose }) => {
+    const [inputValue, setInputValue] = useState('');
+
+    const createStudents = async () => {
+        console.log("inputValue: ",inputValue,"classID: ", classID)
```

Comment:
Is it possible to make console.log only show for us and not clients. Or is it used only during development for testing

Reply:
There is nothing wrong with using console.log when you are debugging, but don't leave this in the PR, as this will go to main branch which is expected to be deployed.  What you print out on the console could end up being very sensitive information.

Saying this I'm basing this on the presumption that this file is run on the client (i.e. front end), not the server (i.e. backend), so I'm expecting that if I open my browser and I create a student I will be able to see the inputValue and classID and so on.

Reply:
Ahh ok thank you